contracts/SpokePool.sol

```diff
@@ -55,13 +55,19 @@ abstract contract SpokePool is
     // refunds and slow relays.
     address public hubPool;
 
+    // Note: storage variables prefixed with DEPRECATED used to be variables that could be set by the cross-domain
+    // admin. Admins ended up not changing these in production, so to reduce gas in deposit/fill functions,
+    // we are converting them to private variables to maintain the  contract storage layout and replacing them with
+    // immutable or constant variables, because retrieving a constant value is cheaper than retrieving
+    // a storage variable. Please see out the immutable/constant variable section.
+
     // Address of wrappedNativeToken contract for this network. If an origin token matches this, then the caller can
     // optionally instruct this contract to wrap native tokens when depositing (ie ETH->WETH or MATIC->WMATIC).
-    WETH9Interface public wrappedNativeToken;
+    WETH9Interface private DEPRECATED_wrappedNativeToken;
```

Comment:
nit: might be useful to replace the comments above this and DEPRECATED_depositQuoteTimeBuffer to explain that these variables are no longer used, but are preserved to keep the slot spacing the same.

Reply:
ah yes, i already copied these comments down to the new immutable/constants var section so i can remove these